tests/test_sshfsmount.cpp

```diff
@@ -241,3 +309,52 @@ TEST_F(SshfsMount, unblocks_when_sftpserver_exits)
     mount.join();
     EXPECT_TRUE(stopped_ok);
 }
+
+TEST_F(SshfsMount, throws_when_unable_to_change_dir)
+{
+    bool invoked{false};
+    auto request_exec = make_exec_that_fails_for({"cd"}, invoked);
+    REPLACE(ssh_channel_request_exec, request_exec);
+
+    EXPECT_THROW(make_sshfsmount(), std::runtime_error);
+    EXPECT_TRUE(invoked);
+}
+
+TEST_F(SshfsMount, throws_when_unable_to_get_current_dir)
+{
+    bool invoked{false};
+    auto request_exec = make_exec_that_fails_for({"pwd"}, invoked);
+    REPLACE(ssh_channel_request_exec, request_exec);
+
+    EXPECT_THROW(make_sshfsmount(), std::runtime_error);
+    EXPECT_TRUE(invoked);
+}
+
+TEST_F(SshfsMount, executes_commands)
```

Comment:
Sorry I'm late to this party, but this test is quite problematic.  I'm working on rebasing #1328 and have added a number of new commands and removed old ones.  This totally breaks this test and since this test is very specific to commands and order, it is not trivial to update it when we make changes elsewhere.   Looking at this test, I'm not sure what this test gains us that is not covered elsewhere.  Is this test really necessary?